project/ws/app/src/lib/routes/home/routes/marketplace-provider/components/market-place-dashboard/market-place-dashboard.component.ts

```diff
@@ -36,62 +38,86 @@ export class MarketPlaceDashboardComponent implements OnInit {
     btnText: string,
     action: string
   }[] = []
+  providersRequestsList = []
+  searchProvider$ = new Subject<string>();
+  searchRegisteredProvider$ = new Subject<string>();
 
   constructor(
     private dialog: MatDialog,
     private router: Router,
     private marketPlaceSvc: MarketplaceService,
     private snackBar: MatSnackBar,
-    private datePipe: DatePipe
+    private datePipe: DatePipe,
+    private loaderService: GlobalEventsService
+
   ) { }
 
   ngOnInit() {
     this.intializeTableData()
+    this.searchProvider$.pipe(
```

Comment:
unsubscibe these in the destroy

Reply:
Merging now for DEV deployment only - @vishnubansaltarento, please check the comments are resolved before next PR merge to higher environments